src/Flex.js

```diff
@@ -1,57 +1,22 @@
 
 import React from 'react'
-import assign from 'object-assign'
-import Base from './Base'
-import config from './config'
+import Reflex from './Reflex'
 
 /**
  * Creates a flexbox context to control layout of children.
  */
 
-const Flex = ({
-  wrap,
-  column,
-  align,
-  justify,
-  auto,
-  ...props
-}, { reflexbox }) => {
-  let display = 'flex'
-  const { breakpoints } = { ...config, ...reflexbox }
+const Flex = ({ is = 'div', className, ...props }) => {
+  const Base = Reflex(is)
```

Comment:
Yeah, I think this will be hoisted up and the `is` prop will just go. I was trying to see if I could support the current API, but this just seems awful